src/util/bip32.rs

```diff
@@ -76,6 +77,17 @@ impl fmt::Debug for ExtendedPrivKey {
             .field("chain_code", &self.chain_code)
             .finish_non_exhaustive()
     }
+    #[cfg(not(feature = "unstable"))]
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("ExtendedPrivKey")
+            .field("network", &self.network)
+            .field("depth", &self.depth)
+            .field("parent_fingerprint", &self.parent_fingerprint)
+            .field("child_number", &self.child_number)
+            .field("chain_code", &self.chain_code)
+            .field("private_key", &"[redacted]")
```

Comment:
Why not just do this all the time? This doesn't seem worth exposing a feature for.

Reply:
This would be the simplest approach without having to detect compiler versions.